src/administration/integrations/blackfire.md

```diff
@@ -69,3 +83,63 @@ Access your site via your browser and click `Profile` in the Blackfire Companion
 ![Blackfire Companion](/images/blackfire-companion.png)
 
 That's it! Your site will be profiled and you should get all the results in your Blackfire account.
+
+## Going further with Blackfire
+
+Blackfire also enables to:
+- collaborate with the rest of your team
+- write performance tests
+- automate profiling with periodic builds
+- integrate further with Platform.sh by enabling to automate profiling as each
+code commit
+- integrate with New Relic for combined benefits of monitoring and profiling
+- integrate with GitHub, Bitbucket and GitLab to show the results of Blackfire
+builds at the commit status level
```

Comment:
Blank line before the start of a list.  Also, most of the Docs use * for bullets instead of -, so let's be consistent here.